### PR TITLE
#65 - Create post-requests for the user registration and authorization

### PR DESCRIPTION
**Pull request by ImmortalIdiot**

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -68,4 +68,8 @@ dependencies {
     androidTestImplementation("androidx.compose.ui:ui-test-junit4")
     debugImplementation("androidx.compose.ui:ui-tooling")
     debugImplementation("androidx.compose.ui:ui-test-manifest")
+    implementation("org.postgresql:postgresql:42.6.0")
+    implementation("com.squareup.retrofit2:retrofit:2.9.0")
+    implementation("com.squareup.retrofit2:converter-gson:2.9.0")
+    implementation("com.squareup.okhttp3:logging-interceptor:4.9.1")
 }
```
Comment (LittlePony00) [inline app/build.gradle.kts:75]: add new line

File: app/src/main/java/com/immortalidiot/rutlead/database/ClientAPI.kt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        if (retrofit == null) {
+            retrofit = Retrofit
+                .Builder()
+                .baseUrl(url)
+                .addConverterFactory(GsonConverterFactory.create())
+                .build()
+        }
+        return retrofit!!
+    }
+}
```
Comment (LittlePony00) [inline app/src/main/java/com/immortalidiot/rutlead/database/ClientAPI.kt:20]: This is strange...

Comment (ImmortalIdiot) [reply app/src/main/java/com/immortalidiot/rutlead/database/ClientAPI.kt:20]: Why?

Comment (ImmortalIdiot) [reply app/src/main/java/com/immortalidiot/rutlead/database/ClientAPI.kt:20]: https://habr.com/ru/articles/520544/
In this article author creates Client just like me

**Review by LittlePony00 — APPROVED** [review]

File: app/build.gradle.kts (hunk 1 above)
Comment (LittlePony00) [reply app/build.gradle.kts:75]: yes

Comment (ImmortalIdiot) [reply app/build.gradle.kts]: Done

**Review by LittlePony00 — APPROVED** [review]

**Pull request merged**